core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableInfo.java

```diff
@@ -79,6 +79,9 @@
   // this idenifier is a lazy field which will be created when it is used first time
   private AbsoluteTableIdentifier identifier;
 
+  // table comment
+  private String tableComment;
```

Comment:
no need to add this attribute as it is already stored in properties

Reply:
Handled it in  https://github.com/apache/carbondata/pull/1472